tests/20-identity-list.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Use two spaces instead of four for indentation in the test scripts.

Reply:
Fixed.